CHANGELOG.rst

```diff
@@ -20,6 +20,7 @@ Fixed
 Changed
 -------
 * Some symbology changed to enable the removal of QgsHighlight due to bugs
+* Metadata updated to reflect additional Tasman and Nelson outlines.
```

Comment:
Data dictionary and metadata updated to reflect Tasman and Nelson building outlines updates.

Reply:
Done